client/my-sites/importer/newsletter/content-upload/importing-pane.jsx

```diff
@@ -257,8 +257,12 @@ export class ImportingPane extends PureComponent {
 						siteId={ siteId }
 						sourceType={ sourceType }
 						sourceAuthors={ customData.sourceAuthors }
-						sourceTitle={ customData.siteTitle || this.props.translate( 'Original Site' ) }
-						targetTitle={ siteName }
+						sourceTitle={
+							sourceType === 'Substack'
+								? this.props.translate( 'Substack' )
+								: customData.siteTitle || this.props.translate( 'Original Site' )
+						}
+						targetTitle={ this.props.translate( 'WordPress.com' ) }
```

Comment:
Not a blocker, but can "Substack" and "WordPress.com" be translated? 🤔

Reply:
Maybe with special characters? But I saw other calls to translate "WordPress.com" in the repo and figured its worth it. I imagine substack is the same.

Reply:
TIL!